dsl/git_push_test.jmx

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="2.8" jmeter="2.13 r1665067">
```

Comment:
This commit mentioned it: https://github.com/alphagov/tsuru-performance-tests/commit/913fe39efe89b25811eda82237ddc458ec2fa789  - happy to remove it if you think it's overkill.